docs/GettingStarted.md

```diff
@@ -669,6 +669,8 @@ To disable a plugin create the file `~/.yard/ignored_plugins` with a list
 of plugin names separated by newlines. Note that the `.yard` directory might
 not exist, so you may need to create it.
 
+You may find some useful YARD plugins on [RubyGems](https://rubygems.org/search?utf8=%E2%9C%93&query=name%3A+yard) or with a [Google avanced querry](https://www.google.com/search?q=site%3Arubygems.org+intitle%3A%22yard-%22+OR+intitle%3A%22yard_%22).
```

Comment:
typo: `querry` -> `query`

also nitpick but consider trying to enforce an 80char line width here. You can use the `[RubyGems][rubygems]` linking format with the references just below.

Reply:
+ my bad for the typo
+ I wanted to but forget about the line of 80 chars